### ISSUE TITLE
foliate executable has unintuitive name

### ISSUE THREAD

**Issue by emacsomancer** [void-linux/void-packages#20714]
### System

* xuname:  
  *Void 5.4.29_1 x86_64 GenuineIntel uptodate hold rrrmDFFFFFFFFFFFFFF*
* package:  
  *affected package(s) including the version*

### Expected behavior
run [Foliate](https://johnfactotum.github.io/foliate/") via `foliate` or similar
### Actual behavior
Foliate must be run as `com.github.johnfactotum.Foliate`
### Steps to reproduce the behavior
Install `foliate`

(I'm not sure the best way of renaming the executable in the [package build](https://github.com/void-linux/void-packages/blob/master/srcpkgs/foliate/template). I tried adding:

```
post_install() {
	vbin com.github.johnfactotum.Foliate foliate
}
```
And some similar variants, but didn't have any luck.)

**Comment by not-chicken** [comment]
Renaming the executable alone won't work. The binary name has been hardcoded mulitiple times in the source.

**Comment by sgn** [comment]
[I'm not the pkg-commiter]
If I were you, I would do like this: https://github.com/sgn/void-packages/commit/d505b965040159558934d37d12b93cc2cc39fdc4

**Comment by Duncaen** [comment]
This assumes nothing else would possibly try to execute this program.
I think keeping the name and adding a symlink to `/usr/bin` is generally a better approach and is done by archlinux.

**Comment by emacsomancer** [comment]
> This assumes nothing else would possibly try to execute this program.
> I think keeping the name and adding a symlink to `/usr/bin` is generally a better approach and is done by archlinux.

This was my local solution.

**Comment by sgn** [comment]
Closed by #20732
